### ISSUE TITLE
Text to Speech always uses the 'Eleven English v1' model (regardless of what model is selected)

### ISSUE THREAD

**Issue by GOvEy1nw** [n8n-ninja/n8n-nodes-elevenlabs#7]
Hey, I'm guessing there's some issue with the model_id variable in the code as I've found that the voice is always processed with the Eleven English v1 model, regardless of what model I select in the node options. It'd be great if this could be fixed please.

**Comment by flowzen1337** [comment]
Agree, same problem here, also if you add it with the Voice ID it still uses the same model as mentioned

**Comment by gnewtonn** [comment]
Yeah same here, fixing this would be great

**Comment by martintomko** [comment]
Same :/

**Comment by GOvEy1nw** [comment]
I ended up fixing it myself and dropping a different package, other than that fix I've found the rest to work fine for what I need, so haven't touched it since: https://www.npmjs.com/package/@govey1nw/n8n-nodes-11labs

**Comment by gnewtonn** [comment]
> I ended up fixing it myself and dropping a different package, other than that fix I've found the rest to work fine for what I need, so haven't touched it since: https://www.npmjs.com/package/@govey1nw/n8n-nodes-11labs

Thanks for sharing this! gonna test it

**Comment by Nasnl** [comment]
> I ended up fixing it myself and dropping a different package, other than that fix I've found the rest to work fine for what I need, so haven't touched it since: https://www.npmjs.com/package/@govey1nw/n8n-nodes-11labs

Oh, that is a relief! Will try that too, since this is not working for me either.

**Comment by Nasnl** [comment]
I guess this is very much related to Issue #3 . Although above and in that issue I see hints that it might be solved, from all these links if I install the custom node, it still gives me English accents only. 
Anyone can explain me what I might do wrong?

**Comment by lvalics** [comment]
Try from here: https://www.npmjs.com/~lvalics 
[n8n-nodes-elevenlabs-enhanced](https://www.npmjs.com/package/n8n-nodes-elevenlabs-enhanced)
